src/Evolution/Systems/Ccz4/TimeDerivative.cpp

```diff
@@ -144,6 +145,10 @@ void TimeDerivative<Dim>::apply(
     const tnsr::ijK<DataVector, Dim>& d_field_b,
     const tnsr::ijkk<DataVector, Dim>& d_field_d,
     const tnsr::ij<DataVector, Dim>& d_field_p) {
+  ASSERT(
+      evolve_shift == EvolveShift::True or evolve_shift == EvolveShift::False,
+      "Unknown Ccz4::EvolveShift.");
+
```

Comment:
Since `Ccz4::EvolveShift` inherits from `bool`, this might be silly and unnecessary

Reply:
I'm not worried about it :)